testing/pipeline.config.js

```diff
@@ -0,0 +1,59 @@
+const { defineConfig } = require("cypress");
```

Comment:
adding this file so that we can use a script added to use this config to hit the static site